examples/standalone/README.md

```diff
@@ -0,0 +1,18 @@
+### Testplane Standalone Browser Usage Example
```

Comment:
Looks like we should add .gitignore in shit folder in order to not commit captured screens